### ISSUE TITLE
Add ability to import wallet to FFS

### ISSUE THREAD

**Issue by ukstv** [textileio/powergate#456]
Currently it is possible to just create new Filecoin wallets. It would be great to have ability to import existing ones into Powergate/Lotus. Anyway, Lotus supports importing wallets.

**Comment by ukstv** [comment]
Do you think it could happen in a month or two? 

Alternatively, could I import a key to Lotus and link it somehow to ffs instance?

**Comment by jsign** [comment]
> Do you think it could happen in a month or two?
> 
> Alternatively, could I import a key to Lotus and link it somehow to ffs instance?

Yeah, we can prioritize that in that time frame. You could import it manually in the Lotus node, but we at the moment only recognize wallet addresses created by the FFS instance. But you can assume we can land this feature if that's important or blocking.

**Comment by ChristianOConnor** [comment]
How is this feature coming along? Is there a beta version of this feature where you can import the wallet into powergate?
